client/src/pages/email-sign-up.tsx

```diff
@@ -115,6 +114,7 @@ function AcceptPrivacyTerms({
   const { t } = useTranslation();
   const acceptedPrivacyRef = useRef(acceptedPrivacyTerms);
   const acceptTermsRef = useRef(acceptTerms);
+  const newAccount = isSignedIn && completedChallengeCount > 1;
```

Comment:
```suggestion
  const newAccount = isSignedIn && completedChallengeCount < 1;
```

I think this is backwards. newAccount should be no challenges completed?

Reply:
> We get one completed challenge count for creating the account? Don't we?

IIRC you get a point, but not a completed challenge.